packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -978,17 +998,16 @@ export function createYAxisLabels(
       text
         .append('tspan')
         .attr('id', 'showDots')
-        .attr('x', isRtl ? 0 : x)
```

Comment:
RTL check not needed? I see y labels are not coming properly in RTL mode. But it can be taken as separate bug.

Reply:
The sole purpose of the createYAxisLabels utility is to truncate labels that exceed the noOfCharsToTruncate limit. So, there's no need to set the x attribute based on the layout direction - the default value works perfectly well.
I’ve reviewed the Storybook examples in RTL mode, and the output appears correct. If it looks off in the plotly_examples app, it’s likely due to an incorrect RTL setup.